From f429f26dc820b971210556cc89cb9465b532f507 Mon Sep 17 00:00:00 2001
From: Linh Nguyen <linh.nguyen@steinsvik.vn>
Date: Wed, 5 Sep 2018 10:25:36 +0200
Subject: [PATCH] Don't build doc

---
 Makefile.am     |  2 +-
 src/Makefile.am | 12 ------------
 2 files changed, 1 insertion(+), 13 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index cf43447..7cd3017 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -2,7 +2,7 @@ ACLOCAL_AMFLAGS=-I m4

 bin_SCRIPTS = img2fwup

-SUBDIRS=src tests docs
+SUBDIRS=src tests
 EXTRA_DIST=README.md \
 	   VERSION \
 	   CHANGELOG.md \
diff --git a/src/Makefile.am b/src/Makefile.am
index ec0713c..70e46c0 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -100,18 +100,6 @@ if ENABLE_GCOV
 fwup_CFLAGS += -fprofile-arcs -ftest-coverage
 endif

-if HAVE_HELP2MAN
-man_MANS = fwup.1
-CLEANFILES = $(man_MANS)
-
-fwup.1: fwup$(EXEEXT) fwup.h2m
-	$(HELP2MAN) --no-discard-stderr --output=$@ --no-info --include fwup.h2m \
-	    -n "configurable embedded firmware update creator and runner" \
-	    ./fwup$(EXEEXT)
-else
-	@echo  "Warning: help2man not available, no man page created."
-endif
-
 if ENABLE_BASH_COMPLETION
 bashcompletiondir = $(BASH_COMPLETION_DIR)
 dist_bashcompletion_DATA = bash-completion/fwup
--
2.7.4
